lang/de.yml

```diff
@@ -180,6 +180,7 @@ de:
     RESPONSE: "Re: %s"
     SHOWLINK: "Zeige Thema"
     SINGULARNAME: "Beitrag"
+    NEWREPLY: "Neue Antwort zu"
```

Comment:
This key isn't even added to the master en.yml. It needs to be added there first, pushed up to https://www.transifex.com/projects/p/silverstripe-forum/language/en/, and translated there. Otherwise the next time translations are synchronised all of these new keys will be destroyed.